great_expectations/expectations/expectation.py

```diff
@@ -2413,10 +2413,10 @@ def validate_configuration(
 
 
 class ColumnMapExpectation(TableExpectation, ABC):
-    map_metric = None
+    map_metric = ""
```

Comment:
This change won't be an issue right? Seems to be a base for a lot of classes

Reply:
I ended up changing this to `map_metric: Optional[str] = None` to make the type checker happy, because you were right that this is `None` vs `str` specific logic in this class.

Reply:
Seems like that `str` vs `None` logic might be wrong.
I would expect an empty string would be invalid.